web/src/components/storage/ProposalResultSection.jsx

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+// @ts-check
+
+import React, { useState } from "react";
+import { Button, Skeleton } from "@patternfly/react-core";
+import { sprintf } from "sprintf-js";
+import { _, n_ } from "~/i18n";
+import { deviceChildren, deviceSize } from "~/components/storage/utils";
+import DevicesManager from "~/components/storage/DevicesManager";
+import { If, Section, Reminder, Tag, TreeTable } from "~/components/core";
+import { ProposalActionsDialog } from "~/components/storage";
+
+/**
+ * @typedef {import ("~/client/storage").Action} Action
+ * @typedef {import ("~/client/storage").StorageDevice} StorageDevice
+ * @typedef {import("~/client/mixins").ValidationError} ValidationError
+ */
+
+/**
+ * Renders information about planned actions, allowing to check all of them and warning with a
+ * summary about the deletion ones, if any.
+ * @component
+ *
+ * @param {object} props
+ * @param {Action[]} props.actions
+ * @param {string[]} props.systems
+ */
+const DeletionsInfo = ({ actions, systems }) => {
+  const total = actions.length;
+
+  if (total === 0) return;
+
+  // TRANSLATORS: %d will be replaced by the amount of destructive actions
+  const warningTitle = sprintf(n_(
+    "There is %d destructive action planned",
+    "There are %d destructive actions planned",
+    total
+  ), total);
+
+  // FIXME: Use the Intl.ListFormat instead of the `join(", ")` used below.
+  // Most probably, a `listFormat` or similar wrapper should live in src/i18n.js or so.
+  // Read https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat
+  return (
+    <Reminder title={warningTitle} variant="subtle">
+      <If
+        condition={systems.length > 0}
+        then={
+          <p>
+            {_("Including the deletion of")} <strong>{systems.join(", ")}</strong>
```

Comment:
The thing is that I'd like to avoid abusing of `dangerouslySetInnerHTML` and that's the reason I did not use a replacement there.

Is it enough for you something like?

```suggestion
            {_("Affecting")} <strong>{systems.join(", ")}</strong>
```

Reply:
Good enough for now, yes.